openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -710,6 +706,45 @@ Cache-Control: no-store
 }
 ```
 
+# Nonce Endpoint {#nonce-endpoint}
+
+A Credential Issuer that requires `c_nonce` values to be incorporated into proofs in the Credential Request (see (#credential-request)) MUST offer a Nonce Endpoint. This endpoint allows a Client to acquire a fresh `c_nonce` value without the overhead of a full Credential Request.
+
+The `nonce_endpoint` Credential Issuer Metadata parameter, as defined in (#credential-issuer-parameters), contains the URL of the Credential Issuer's Nonce Endpoint.
```

Comment:
```suggestion
The Credential Issuer indicates support for this endpoint using `nonce_endpoint` Credential Issuer Metadata parameter, as defined in (#credential-issuer-parameters), which contains the URL of the Credential Issuer's Nonce Endpoint.
```
I was not sure what is the purpose of the original sentence...

Reply:
to say where the actual endpoint value can be found